src/api/models/Activity.ts

```diff
@@ -60,4 +113,44 @@ Activity.init(
     }
 );
 
-module.exports = Activity;
+const Activity_Genres = sequelize.define(
+    'Activity_Genres',
+    {
+        id: {
+            type: DataTypes.INTEGER,
+            primaryKey: true,
+            autoIncrement: true,
+            allowNull: false
```

Comment:
Do you have to set allowNull for the id since its the primary key? I think it's implied already